ROLES.md

```diff
@@ -47,14 +44,17 @@ Further, you may self-nominate to be an approver by opening an issue with a list
     <td>
         <p>Member of the GitHub Istio organization</p>
         <p>Edit permission on the Istio Team drive</p>
+        <p>Write permissions on the Istio repos, allowing issues to be manipulated.</p>
     </td>
   </tr>
 
   <tr>
-    <td><a href="#approver">Approver</a></td>
+    <td><a href="#maitainerr">Maintainer</a></td>
     <td>Approve contributions from other members</td>
     <td>Highly experienced and active reviewer and contributor to an area</td>
-    <td>‘approver’ entry in one or more OWNERS file in GitHub (istio/istio.io excluded)</td>
+    <td>Like a member, plus:
+        <p>Able to approve code changes in GitHub</p>
```

Comment:
+1 to this.

For additional considering:  have voting rights in the appropriate working group (e.g,. pilot maintainers would have voting rights in the networking working group)

Reply:
Added